src/main/java/org/o3project/odenos/component/aggregator/Aggregator.java

```diff
@@ -252,6 +252,9 @@ protected void onConnectionChangedAdded(
       }
       orgNwIf = networkInterfaces().get(orgNetworkId.get(0));
       aggNwIf = networkInterfaces().get(networkId);
+    } else {
+      log.error("Unexpected network type: {}", type);
+      throw new IllegalArgumentException("Unexpected network type: " + type);
     }
     // Update conversionTable.
     conversionTable().addEntryNetwork(
```

Comment:
Intention of this patch is to avoid NPE by `orgNwIf#getNetworkId()` in the line right after here.  ([L258](https://github.com/o3project/odenos/blob/1046b5557745e09e18935e15530f3c64eb658911/src/main/java/org/o3project/odenos/component/aggregator/Aggregator.java#L258) in old file, L261 in new file)